src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IArgument.cs

```diff
@@ -2018,7 +2018,7 @@ public void M2()
   Arguments(1):
       IArgument (ArgumentKind.DefaultValue, Matching Parameter: s) (OperationKind.Argument) (Syntax: 'p.M1()')
         IConversionExpression (Implicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: System.Int32) (Syntax: 'p.M1()')
-          Conversion: CommonConversion (Exists: True, IsIdentity: True, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
+          Conversion: CommonConversion (Exists: False, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
```

Comment:
I think what's going on here is that, since the IL is converted to a metadata reference, that's where the error actually is. When I inspected the compilation in the debugger, it indicated that this syntax has no errors, so the verifier will not print that the node IsInvalid. However, the conversion generated for the default parameter does not exist, which we see here.